ci-operator/step-registry/ipi/conf/vsphere/check/ipi-conf-vsphere-check-commands.sh

```diff
@@ -57,6 +54,19 @@ else
 
 fi
 
+#to support vsphere-ipi-zones-multisubnets-external-lb-f28 and vsphere-ipi-zones-multisubnets-external-lb-f28-destructive.add these var to specify the portgroups.
+if [[ -n "${VSPHERE_MULTIZONE_LEASED_RESOURCE:-}" ]]; then
+    i=0
+    for multizone_leased_resource in ${VSPHERE_MULTIZONE_LEASED_RESOURCE}; do    
+	 multizone_vlanid=$(awk -F. '{print $3}' <(echo "${multizone_leased_resource}"))
+	 multizone_portgroup="ci-vlan-${multizone_vlanid}"
+	 i=$((i + 1))
+	 portgroup_list[$i]="${multizone_portgroup}"
```

Comment:
If you want `portgroup_list` to be an array var, then you can do things like this to append elements:
```
portgroup_list=()
...
portgroup_list+=("${vsphere_portgroup}")
...
portgroup_list+=("${multizone_portgroup}")
...
for i in "${portgroup_list[@]}"; do
....
done
```